.idea/misc.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="JavaScriptSettings">
+    <option name="languageLevel" value="ES6" />
+  </component>
+</project>
```

Comment:
Not sure what this is? please explain

Reply:
files auto-generated by daniel's jetbrain IDE, will remove as he is now using atom